plat/rpi/rpi5/include/rpi5_smc.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef PLAT_RPI_RPI5_INCLUDE_RPI5_SMC_H_
+#define PLAT_RPI_RPI5_INCLUDE_RPI5_SMC_H_
+
+#include <lib/smccc.h>
+
+/*
+ * Process scmi message pending in the SCMI message
+ * shared memory buffer.
+ */
+#define RPI5_SIP_SMC_SCMI 0x82000002
+
+/*
+ * Error codes for SMC calls
+ */
+#define RPI5_SMC_OK			SMC_OK
+#define RPI5_SMC_NOT_SUPPORTED		SMC_UNK
```

Comment:
It is better to add a separate "not supported" error code instead of using `SMC_UNK`

Reply:
According to the smccc.h from ARM-tf there is another error code SMC_ARCH_CALL_NOT_SUPPORTED which equals to SMC_UNK. Also, from the linux kernel side there is SMCCC_RET_NOT_SUPPORTED which is also -1. I thinks I will make 
`#define RPI5_SMC_NOT_SUPPORTED		SMC_ARCH_CALL_NOT_SUPPORTED `
to make code more readable still leaving default error codes.

Reply:
As  I said below, there is a difference between "unknown call" and "not supported call". In your case this is clearly "unknown call".

Reply:
removed this error code at all after rethinking